### PR TITLE
(Planner 1): add mechanism to change between m/min and ft/min 

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Add code to qml, to start testing online change of METERS <-> FEET.

### CODE REVIEW

File: mobile-widgets/qml/DivePlannerSetup.qml [hunk 1]
```diff
@@ -10,6 +10,7 @@ import org.kde.kirigami 2.4 as Kirigami
 Kirigami.ScrollablePage {
 	title: qsTr("Dive planner setup")
 
+	property string textLength: (PrefUnits.length === "meters") ? qsTr("m/min") : qsTr("ft/min")
```
Comment (dirkhh) [inline mobile-widgets/qml/DivePlannerSetup.qml]: I an not in love with the name of that property. Wouldn't this be `speedUnit` or something along those line?

File: mobile-widgets/qml/DivePlannerSetup.qml [hunk 2]
```diff
@@ -10,7 +10,14 @@ import org.kde.kirigami 2.4 as Kirigami
...
 	title: qsTr("Dive planner setup")
 
-	property string textLength: (PrefUnits.length === "meters") ? qsTr("m/min") : qsTr("ft/min")
+	property string textLength: (PrefUnits.unit_system === "metric") ? qsTr("m/min") : qsTr("ft/min")
+//	property string textLength: PrefUnits.unit_system
+	onTextLengthChanged: {
+		console.log("--> New text << " + textLength )
+		spinAscrate75.value++
+		spinAscrate75.value--
+	}
```
Comment (dirkhh) [inline mobile-widgets/qml/DivePlannerSetup.qml:26]: I don't understand what you are trying to do here. The increment/decrement looks like this is test code?

**Review by dirkhh — CHANGES_REQUESTED** [review]
Two comments, one on the property name, one questioning if this is production code :-)

File: mobile-widgets/qml/DivePlannerSetup.qml (hunk 1 above)
Comment (janiversen) [reply mobile-widgets/qml/DivePlannerSetup.qml]: changed

File: mobile-widgets/qml/DivePlannerSetup.qml (hunk 2 above)
Comment (janiversen) [reply mobile-widgets/qml/DivePlannerSetup.qml:26]: it is not test code, added a larger comment as to why.

**Review by dirkhh — APPROVED** [review]
that ++/-- is one of those things where my jaw just hits the floor. QML is a strange beast for sure.
Thanks for the explanation

**Comment by dirkhh** [comment]
I did some small rework of the whole series and pushed it to master.

**Pull request closed without merging**